runtime/doc/api.txt

```diff
@@ -2791,6 +2791,8 @@ nvim_buf_set_extmark({buffer}, {ns_id}, {line}, {col}, {*opts})
                   • url: A URL to associate with this extmark. In the TUI, the
                     OSC 8 control sequence is used to generate a clickable
                     hyperlink to this URL.
+                  • scoped: boolean that indicates that the extmark should
+                    only be displayed in the namespace scope. (experimental)
```

Comment:
This extra flag seems kind of random. It says "ignore the scopes for the given namespace". If you don't want an extmark to obey the properties of its namespace, why not create a separate namespace ?

Reply:
If opting out is now allowed, that means an extmark will be invisible until its namespace is added to a window, which is undesirable.

Reply:
> an extmark will be invisible until its namespace is added to a window

Why must it be like that? If a namespace has no "window scopes", can't we assume it has "all windows" scope?